docs/journalist/starting_client.rst

```diff
@@ -1,16 +1,66 @@
-Starting the SecureDrop client
+Starting the SecureDrop Client
 ==============================
 
 .. include:: ../includes/top-warning.rst
 
+Double-click on the Desktop shortcut to launch the SecureDrop Client
+application.
+
+.. | screenshot_desktop-shortcut|
+
 Performing updates
 ------------------
 
+When you launch the application, it will first check for important security
+updates. This can take several minutes.
+
+.. | screenshot_check_updates |
+
+If updates are found, you will be asked to apply them before launching the
+client. You have the option of completing this now, or at another time. Keep in
+mind that updates are important to security, and you should apply them the same
+day you receive the notification.
+
+It is important that, when you do click "Apply Updates," you allow the update
+process to complete fully, without closing or interrupting it. The update
+process may take some time.
+
+.. | screenshot_apply_updates |
+
+Once the update process has been completed, the SecureDrop client will launch.
+
 Signing in
 ----------
 
+To sign in, enter the username and passphrase provided to you by your
+SecureDrop administrator, as well as the 2-factor method you have set up (such
+as Google Authenticator or FreeOTP). If you have used SecureDrop before, these
+are the same credentials that you would use to log in to the Journalist
+Interface.
+
+.. | screenshot_sd-app_login |
+
+After signing in, you will be prompted by a dialog that says “Do you allow VM
+‘sd-app’ to access your GPG keys (now and for the following 28800 seconds)?”.
+Click **Yes.** This dialogue may appear immediately after signing in, or when
+you click on a source submission.
+
+.. | screenshot_gpg_access |
+
 Troubleshooting tips
 ~~~~~~~~~~~~~~~~~~~~
 
+[This section is a stub]
+ - Check your network connectivity if you are having issues with the updater or
+   with signing in.
+ - You may want to refer to the Qubes documentation to understand the layout,
+   toolbars, and menu options you see in Qubes.
+
+
 Working offline
 ---------------
```

Comment:
Let's explain here why the user doesn't need to log in to see previously downloaded content.

Reply:
How do you feel about what I added in 437ad10 by way of explanation? It's still not at great length; if you had in mind more detail here happy to add.

Reply:
LGTM!